openj9.test.sharedClasses/src/test.sharedClasses/net/openj9/stf/SharedClasses.java

```diff
@@ -211,7 +216,8 @@ public void setUp(StfCoreExtension test, StfSharedClassesExtension sharedClasses
 						.addProjectToClasspath("openj9.test.sharedClasses")
 						.runClass(JavaGen.class)
 						.addArg(sharedClassesDataDir.getSpec())
-						.addArg("10000"));
+						.addArg("10000")
+						.addArg(String.valueOf(javaVersion)));
```

Comment:
Do we need to add `javaVersion` as an arg?

Reply:
I am passing the `javaVersion` as an argument to be used by J`avaGen.class` file, or should we set it as environment Variable or as a static field ?